jest.config.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+  testMatch: ['<rootDir>/packages/**/test/**/*.{js,ts}'],
```

Comment:
Can change this to

```  testMatch: ['<rootDir>/packages/slate-react/test/**/*.{js,ts,tsx,jsx}'],```

Reply:
Thx @anishagg17. I have updated this PR based on your input.